### PR TITLE
Fix typos in function parameter and comments

### PR DESCRIPTION
**Pull request by alexandear**
Fix typos in comments and funtion's parameter:
- necssary - necessary
- deserialised - deserialized

### CODE REVIEW

File: tests/serialization/test_dag_serialization.py [hunk 1]
```diff
@@ -523,7 +523,7 @@ def validate_deserialized_task(
         if isinstance(task.params, ParamsDict):
             assert serialized_task.params.dump() == task.params.dump()
 
-        # Check that for Deserialised task, task.subdag is None for all other Operators
+        # Check that for Deserialized task, task.subdag is None for all other Operators
```
Comment (kaxil) [inline tests/serialization/test_dag_serialization.py:526]: Both the spellings are correct: serialization (US spelling) or serialisation (UK spelling)

Comment (potiuk) [reply tests/serialization/test_dag_serialization.py:526]: Should we standardise (or rather standardize :D)  on the US spelling ? I think this is what we mostly have in our spelling_wordlist and docs :).

Comment (alexandear) [reply tests/serialization/test_dag_serialization.py:526]: @kaxil  I searched across the repository:
- `deserialis*` - 4 matches
- `deserializ*` - 100+ matches
It's better to choose one mostly used spelling.

Comment (kaxil) [reply tests/serialization/test_dag_serialization.py:526]: I think there is no need to standardi(s/z)e this tbh :D . Both should be accepted - we gain nothing by this standardization.

**Review by kaxil — APPROVED** [review]

**Pull request merged**